pagopa/pn-frontend
# PN-11770 - Multi language integration on Footer

## Short description
Multi language integration on Footer

## List of changes proposed in this pull request
- Footer 

## How to test
Tra to change language from footer, refresh page anche check that footer has right language selected
